src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -406,5 +406,40 @@ internal static TypeSymbol GetTypeOrReturnType(this Symbol symbol)
                     throw ExceptionUtilities.UnexpectedValue(symbol.Kind);
             }
         }
+
+        internal static bool IsWellKnownTypeIsConst(this ISymbol symbol)
```

Comment:
>this ISymbol symbol [](start = 52, length = 19)

Does it have to be an extension on ISymbol? Could it extend INamedTypeSymbol instead?